src/app/datagrid/hide-show-columns/hide-show.html

```diff
@@ -0,0 +1,87 @@
+<!--
+  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Not a big deal for this PR but in the future we are doing copyright 2016-2017 on files...